include/screencast.h

```diff
@@ -0,0 +1,18 @@
+#ifndef SCREENCAST_H
+#define SCREENCAST_H
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <sys/mman.h>
+#include <pthread.h>
+
+#include "pipewire_screencast.h"
+#include "screencast_common.h"
+#include "wlr_screencast.h"
+#include "xdpw.h"
+
+#endif
```

Comment:
Style: please add a trailing newline to all files.

Reply:
Fixed